### DISCUSSION TITLE
Jamulus with Rode NT USB and Windows 11

### DISCUSSION THREAD

**Discussion by schoeni7** [jamulussoftware/jamulus#3307] [Technical Support & Help]
When I use Jamulus with a laptop mic and headphones, my latency is ok (30-40ms), but the sound quality is very poor. 

That's why I got the Rode NT USB, which I've used before on a Mac with Jamulus and it worked great. 

Unfortunately, the latency on my Windows 11 computer is now very high (40-60 ms) and the connection is also very unstable. I have already changed all the settings, but it is not smooth or stable. Is this due to Asio4All or Windows 11? Friends of mine also use Jamulus with Windows and Asio4All and it works perfectly. 

I'm getting desperate because unfortunately I can't make music this way. :(

I have already tried: 

- Different Rode Microphones
- Download the newest Version of Asio4all and jamulus
- different buffer sizes
- Of course I have a LAN-Connection, not WLAN

Can anyone help me?

**Comment by henkdegroot** [comment]
It seems there is a Rode NT USB asio driver available. Have you tried that instead of asio4all?

**Reply by schoeni7** [reply]
Nope there is no asio driver für Rode NT USB available or where have you found it?

**Reply by henkdegroot** [reply]
This perhaps https://help.rode.com/hc/en-us/articles/7416681704463-NT1-5th-Gen-ASIO-Driver

**Reply by uazure** [reply]
https://rode.com/en-int/products/nt1-5th-generation#section-specs
has the link to it:
<img width="505" height="249" alt="image" src="https://github.com/user-attachments/assets/431fb4dd-af37-4a12-b38e-eb6f0055da8b" />
Here's the link that's there today:
https://edge.rode.com/zip/page/2125/modules/7410/RODE_NT1_5thGen_ASIO.zip

**Comment by ann0see** [comment]
Often ASIO4All is an issue unfortunately. Therefore either use a device with native ASIO support or switch your OS. 
You can also switch to a Ubuntu Studio live disc.

**Comment by pcar75** [comment]
> (...) friends of mine also use Jamulus with Windows and Asio4All (...)

Are your friends using Windows 11 ?  Personnaly, since the MS blurb for 11 felt a lot like for Vista, I'll skip that version or wait after a few upgrades.
On my pc tower, I've installed Ubuntu Studio as dual boot and can shave off anywhere from 5 to 15 ms of overall delay compared to Windows 10Pro (note that distances in the Americas are more important than in Europe).

**Comment by dcorson-ticino-com** [comment]
If you want to stay with Windows you will need to find the ASIO driver for your mic or switch to an audio interface that has an ASIO driver.
ASIO4ALL is a crutch that only sometimes works.  Personally I have never gotten it to work at all on my Win10 Lenovo laptop.
